docs/run_with_docker.md

```diff
@@ -85,7 +85,7 @@ docker run -p 6040:6040 \
 	-e LISK_HOST=<LISK_NODE_IP> \
 	-e LISK_PORT=<LISK_NODE_PORT> \
 	-e REDIS_HOST=lisk-redis \
-	-e FREEGEOIP_HOST=lisk-freegeoip \
+	-e FREEGEOIP_HOST=geoip.lisk.io \
```

Comment:
Missing `-e FREEGEOIP_PORT=80` (the default value is `8080` and wouldn't work anymore)

Reply:
Ok. Fixed with https://github.com/LiskHQ/lisk-explorer/pull/932/commits/22d825a56b849c79b31cc917f35d3a32f372f52c